tests/test_wallpaper.py

```diff
@@ -0,0 +1,14 @@
+import unittest
+from uiplib.Wallpaper import Wallpaper
+import tempfile
+
+
+class WallpaperTest(unittest.TestCase):
+
+    def test_wallpaper(self):
+        wallpaper = Wallpaper()
+        current_wallpaper = wallpaper.get()
+        with tempfile.NamedTemporaryFile(mode="wb") as temp_wallpaper:
+            wallpaper.set(temp_wallpaper.name)
+            self.assertEqual(wallpaper.get(), temp_wallpaper.name)
```

Comment:
so for macOs `wallpaper.get()` returns a set right?, while `temp_wallpaper.name()` returns a string, how does that work out?